{
    "config": {
        "abort": {
            "already_configured": "Dispositivo j\u00e1 configurado"
        },
        "error": {
            "cannot_connect": "A liga\u00e7\u00e3o falhou",
            "invalid_auth": "Autentica\u00e7\u00e3o inv\u00e1lida",
            "invalid_auth_key": "Credenciais de API rejeitadas pela Coinbase devido a uma chave de API inv\u00e1lida.",
            "invalid_auth_secret": "Credenciais de API rejeitadas pela Coinbase devido a um segredo de API inv\u00e1lido.",
            "unknown": "Erro inesperado"
        },
        "step": {
            "user": {
                "data": {
                    "api_key": "Chave da API",
                    "api_token": "Segredo da API"
                }
            }
        }
    },
    "options": {
        "error": {
            "unknown": "Erro inesperado"
        }
    }
}